test/clj_cbor/test_utils.clj

```diff
@@ -148,7 +151,7 @@
 
 (defmacro with-codec
   [opts & body]
-  `(binding [*test-codec* (cbor/cbor-codec ~@(flatten (seq opts)))]
+  `(binding [*test-codec* (apply cbor/cbor-codec (flatten (seq ~opts)))]
```

Comment:
This was done because if you passed in something like `(f)` as part of opts, this it would flatten out that to just `f` without evaluation because it's a macro, I think the semantics you want is to evaluate `opts` on its own and flatten the result.

Reply:
`cbor-codec` accepts a map, so the right way to do this should just be  `(cbor/cbor-codec ~opts)` now.